src/request.ts

```diff
@@ -1,27 +1,33 @@
 import Config from './configuration';
 import colorize from './util/colorize';
+import cloneDeep from './util/clonedeep';
+
+// RequestInit is the type expected by `fetch()` API. 
+export interface FetchOptions extends RequestInit {
+  jwt? : string | undefined
```

Comment:
So if the method returned something *in addition* to `jwt`, would typescript complain?